tests/tests.h

```diff
@@ -67,6 +67,45 @@ using namespace dealii;
 // ------------------------------ Utility functions used in tests -----------------------
 
 /**
+ * Go through the input stream @p in and filter out binary data for the key @p key .
+ * The filtered stream is returned in @p out.
+ */
+void filter_out_xml_key(std::stringstream &in, const std::string &key, std::ostream &out)
```

Comment:
It doesn't have to be a `stringstream` for the first argument -- `std::istream` is good enough.

Reply:
@bangerth fixed.